PatientRegistrationApp/src/components/HouseholdContactInfo.vue

```diff
@@ -1,113 +1,99 @@
 			</v-col>
-			<v-col cols="12" sm="6" md="3">
+			<v-col cols="1" sm="1" md="1" lg="1">
+			</v-col>
+			<v-col cols="12" sm="6" md="6" lg="3">
 				<v-text-field
+					v-model="secondaryPhoneNumber"
+					:disabled="!primaryPhoneNumber"
 					:rules="[v => v.length === 13 || 'Phone number must be 10 digits']"
-					placeholder="(###)###-####"
 					v-mask="'(###)###-####'"
 					label="Secondary Phone Number"
-					v-model="secondaryPhoneNumber"
-					v-show="phoneNumberAvailable"
-				></v-text-field>
+					prepend-icon="mdi-phone">
+				</v-text-field>
 			</v-col>
-			<v-col class="d-flex" cols="4" sm="2">			
+			<v-col cols="11" sm="5" md="5" lg="2">
 				<v-select
 					v-model="secondaryPhoneNumberType"
-					:items="phonetype"
+					:disabled="!primaryPhoneNumber"
+					:items="phoneTypeOptions"
 					label="Phone Type"
-					v-show="phoneNumberAvailable"
+					prepend-icon="mdi-blank"
 				></v-select>
 			</v-col>
 		</v-row>
-		
-		<v-row>
-		<v-radio-group
-        required
-        :rules="[v => !!v || 'This field is required']"
-        v-model="emailAvailableRadioButtons"
-		>
-		<v-col align="right" cols="12">
-			<v-radio label="I have an E-mail address" value="yes" @change="EmailAvailable()"></v-radio>
-			<v-radio label="I do not have an E-mail address" value="no" @change="EmailNotAvailable()"></v-radio>
-        </v-col>
-	</v-radio-group>
-    </v-row>
-	
 		<v-row>
-			<v-col cols="12" sm="12" md="6">
+			<v-col cols="11" sm="5" md="5" lg="5">
 				<v-text-field
-					required
-					:rules="emailRules"
 					v-model="primaryEmail"
-					v-show="emailAvailable"
-					prepend-icon="mdi-menu-right">
-						<template #label>
-						<span class="red--text"><strong>* </strong></span>E-mail Address
-						</template>
-				></v-text-field>
+					:disabled="!permission"
+					:rules="emailRules"
+					prepend-icon="mdi-email"
+					label="Primary E-mail Address">
+				</v-text-field>
+			</v-col>
+			<v-col cols="1" sm="1" md="1" lg="1">
 			</v-col>
-			<v-col cols="12" sm="12" md="6">
+			<v-col cols="11" sm="5" md="5" lg="5">
 				<v-text-field
-					:rules="emailRules"
-					label="Secondary E-mail Address"
 					v-model="secondaryEmail"
-					v-show="emailAvailable"
-				></v-text-field>
+					:disabled="!primaryEmail"
+					:rules="emailRules"
+					prepend-icon="mdi-email"
+					label="Secondary E-mail Address">
+				</v-text-field>
 			</v-col>
 		</v-row>
-
-		<v-row><v-col cols="12" sm="6" md="6">
-			<v-radio-group
-				required
-				:rules="[v => !!v || 'This field is required']"
-				v-model="approval"
-				prepend-icon="mdi-menu-right">
-					<template #label>
-					<span class="red--text"><strong>* </strong></span>May we contact you regarding follow up vaccination information
-					</template>
-				<v-col align="right" cols="3" sm="3" md="3">
-					<v-radio label="Yes" value="yes"></v-radio>
-					<v-radio label="No" value="no"></v-radio>
-				</v-col>
-			</v-radio-group></v-col>
+		<v-row>
+			<p></p>
 		</v-row>
```

Comment:
I don't see a difference with this being here.

Reply:
Deleted